karapace/config.py

```diff
@@ -35,7 +35,7 @@
     "consumer_request_timeout_ms": 11000,
     "consumer_request_max_bytes": 67108864,
     "consumer_idle_disconnect_timeout": 0,
-    "fetch_min_bytes": -1,
+    "fetch_min_bytes": 1,
```

Comment:
This is correct, valid values are `> 0` and default is `1`.
https://kafka.apache.org/documentation/#consumerconfigs_fetch.min.bytes